src/steamHandler.h

```diff
@@ -20,20 +20,6 @@ void checkSteamSwitch() {
             if (steamSwitchReading == LOW && !steamFirstON) {
                 steamON = 0;
             }
-
```

Comment:
Is this really related to brew detection? It looks like this is steam mode handling for quickmill machines